src/models/contact.js

```diff
@@ -10,7 +10,7 @@ class EmailAddress extends RestfulModel {
     return json;
   }
 }
-EmailAddress.collectionName = 'email_addresses';
+EmailAddress.collectionName = 'emails';
```

Comment:
Does this need to change? What are the implications of changing a `collectionName`?

Reply:
I think the `collectionName` is only referenced in the restful-model file for the `save` and `get` methods in order to make API calls to various endpoints. 
At the end of the day, though, since this collection is an attribute of the Contact model, I don't think it will ever be called and the naming ultimately doesn't matter. All other collection names on the Contact model match our API though so I'm more in favor of changing it. However, I'll change it back to `email_addresses` just in case I'm wrong and it is being used somewhere.